andela/odin-ah-frontend
# #160452434 User reset password

#### What does this PR do?
This PR Adds reset password functionality to the codebase.
#### Description of Task to be completed?
- Add password reset form
- Adds passwordRestReducer reducer
- Adds actionType for resetPassword
- Adds resetPassword actions
- Adds resetPassword SCSS style

#### How should this be manually tested?


- Fetch and checkout to the branch
```$xslt
> git fetch
> git checkout  ft-160452434-reset-password-functionality
> npm install 
```
```$xslt
>  Rename the `.env.sample` file on the root directory of the cloned repo to `.env`. 

> Uncomment REACT_APP_HOST which is the link to the frontend application.

> REACT_APP_BASE_API_URL which is the base URL for the backend api. 

For example, `https://odin-ah-backend-staging.herokuapp.com/api/v1`.

> start the dev server with: `npm start`

>  run localhost/**/ ` in your browser and click on the `submit ` and check your email for reset password link.
```
#### Any background context you want to provide?
Nil 
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/160452434

#### Screenshots (if appropriate)
<img width="860" alt="reset password" src="https://user-images.githubusercontent.com/29021978/47926805-cd58bf80-dec1-11e8-9856-57f8b9a3866d.png">

#### Questions:
Nil
